### PR TITLE
merge dev_0.7.0 to main

### PR DESCRIPTION
**Pull request by herbberg**

### CODE REVIEW

File: ugt_fwtools/simulation.py [hunk 1]
```diff
@@ -62,7 +62,7 @@
 DefaultUgtUrl: str = "https://github.com/cms-l1-globaltrigger/mp7_ugt_legacy.git"
 """Default URL for ugt FW repo."""
 
-DefaultUgtTag: str = "v1.22.3"
+DefaultUgtTag: str = "v1.26.0"
```
Comment (arnobaer) [inline ugt_fwtools/simulation.py:65]: `v1.27.0` for simulation too?

File: ugt_fwtools/build_report.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 import os
 from datetime import datetime
 
-ALL_FORMATS = ["markdown", "textile"]
+ALL_FORMATS = ["markdown", "textfile"]
```
Comment (arnobaer) [inline ugt_fwtools/build_report.py:11]: **Note:** key `textile` is not a type but refers to the [textile](https://textile-lang.com/) markup language.

Comment (arnobaer) [reply ugt_fwtools/build_report.py]: fixed in 8a20538

File: ugt_fwtools/simulation.py (hunk 1 above)
Comment (herbberg) [reply ugt_fwtools/simulation.py]: updated to v1.28.0

File: ugt_fwtools/build_report.py (hunk 2 above)
Comment (herbberg) [reply ugt_fwtools/build_report.py]: ok

**Pull request merged**